extension/src/popup/helpers/parseTransaction.ts

```diff
@@ -17,7 +17,11 @@ export const decodeMemo = (memo: any): { value: string; type: MemoType } => {
 
   return {
     value: _memo.value
-      ? buffer.Buffer.from(_memo.value).toString(decodeMethod)
+      ? buffer.Buffer.from(
+          typeof _memo.value === "string"
+            ? Buffer.from(_memo.value, "utf-8")
```

Comment:
Hm, do you have a sense of why we need to do this? Buffer.from should be able to handle a string, right?

Also, should it be `buffer.Buffer.from`? Curious why we're just using the node Buffer instead of the browserify module here